src/Network/Network/Models/PSDdosProtectionPlan.cs

```diff
@@ -26,7 +26,7 @@ public class PSDdosProtectionPlan : PSTopLevelResource
 
         public List<PSResourceId> VirtualNetworks { get; set; }
 
-        public List<PSResourceId> PublicIpAddresses { get; set; }
+        public List<PSResourceId> PublicIPAddresses { get; set; }
```

Comment:
Renaming PublicI**p**Addresses to PublicI**P**Addresses is not allowed in regular release, which is regarded as a breaking change. Please revert this change.

Reply:
we can wait till the next breaking change release for this PR - we want to rename this property to match the swagger change we made in this swagger PR.  https://github.com/Azure/azure-rest-api-specs/pull/21300 - so it's more of  abug fix rather than a breaking change since it never worked in the first place

Reply:
We have workaround to avoid breaking change for this case if you want to map PublicIPAddresses in swagger to PublicIpAddresses in PSDdosProtecyionPlan, so that this property would be work after then. Here is an example:
https://github.com/Azure/azure-powershell/blob/534cdbb7fb9c6f05c089e7ffd6c5e425abd69eb2/src/Network/Network/Common/NetworkResourceManagerProfile.cs#L1446-L1449

Also we are fine if you are good to wait next breaking change.